src/screens/Schedule.js

```diff
@@ -2,6 +2,8 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import styled from 'styled-components';
 import { StyleSheet, Text, View } from 'react-native';
+import CustomButton from "../components/button";
+import AddCustomButton from "../components/AddButton";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8247201)
```suggestion
import AddCustomButton from '../components/AddButton';
```